recipes/libzip/all/conanfile.py

```diff
@@ -0,0 +1,86 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class LibZipConan(ConanFile):
+    name = "libzip"
+    description = "A C library for reading, creating, and modifying zip archives"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/nih-at/libzip"
+    license = "BSD-3-Clause"
+    topics = ("conan", "zip", "libzip", "zip-archives", "zip-editing")
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_lzma": [True, False],
+        "with_bzip2": [True, False],
+        "with_openssl": [True, False],
```

Comment:
Would adding a `with_zstd` option be possible?

Reply:
It is done. I'll push it with crypto fix